static/styles/zulip.scss

```diff
@@ -33,7 +33,7 @@ html {
 }
 
 body {
-    font-family: 'Source Sans Pro', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+    font-family: "Source Sans Pro", "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
There are a lot of formatting-only changes mixed in with this commit, can you remove them?